test/suites/ubuntu-core-edgex-image/test.sh

```diff
@@ -1,39 +1,37 @@
-#!/bin/bash -e
+#!/bin/bash -ex
 
 SSH_USER="$USER"
 SSH_PORT="8022"
 
-printf "# Install curl on host:\n"
-snap install curl; echo
+# Install curl on host
+snap install curl; echo -e "\n"
```

Comment:
Making the output pretty at the cost of added readability to the code is not needed here. Keeping the code clean and easy to read is more important.

Most `; echo -e "\n"`s are unnecessary.